src/docs/product/accounts/quotas/index.mdx

```diff
@@ -7,19 +7,19 @@ redirect_from:
 description: "Learn about what counts towards your quota and how to manage your quota."
 ---
 
-Events and quotas are interconnected in Sentry. At the most basic, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events - errors, attachments, and transactions - to be tracked. Each of these type of events has a quota. When Sentry accepts an event, it counts toward your quota for that type of event.
+Events and quotas are interconnected in Sentry. At the most basic level, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events — errors and transactions — to be tracked. Each of these has a quota, and when Sentry accepts an event, it counts toward your quota for that type of event. We also track and provide a quota for for attachments.
```

Comment:
```suggestion
Events and quotas are interconnected in Sentry. At the most basic level, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events — errors and transactions — and attachments to be tracked. Each of these has a quota, and when Sentry accepts an event, it counts toward your quota for that type of event. We also track and provide a quota for for attachments.
```

Reply:
Just to make clearer that attachments aren't events but they're related to them:
```suggestion
Events and quotas are interconnected in Sentry. At the most basic level, when you [subscribe to Sentry](https://sentry.io/pricing/), you pay for the number of events — errors and transactions — and related attachments to be tracked. Each of these has a quota, and when Sentry accepts an event, it counts toward your quota for that type of event. We also track and provide a quota for for those attachments.
```

Reply:
Yes, that makes it clearer.